File: src/main/java/com/demonwav/mcdev/insight/generation/GenerateEventListenerAction.java
/*
 * Minecraft Dev for IntelliJ
 *
 * https://minecraftdev.org
 *
 * Copyright (c) 2016 Kyle Wood (DemonWav)
 *
 * MIT License
 */

package com.demonwav.mcdev.insight.generation;

import com.intellij.codeInsight.generation.actions.BaseGenerateAction;

public class GenerateEventListenerAction extends BaseGenerateAction {
    public GenerateEventListenerAction() {
        super(new GenerateEventListenerHandler());
    }
}

File: src/main/kotlin/com/demonwav/mcdev/insight/generation/GenerateEventListenerAction.kt
/*
 * Minecraft Dev for IntelliJ
 *
 * https://minecraftdev.org
 *
 * Copyright (c) 2019 minecraft-dev
 *
 * MIT License
 */

package com.demonwav.mcdev.insight.generation

import com.intellij.codeInsight.generation.actions.BaseGenerateAction

class GenerateEventListenerAction : BaseGenerateAction(GenerateEventListenerHandler())
